ontology/yaml/resources/SAFETY/entity_types/FACP.yaml

```diff
@@ -42,6 +44,18 @@ FACP_PAS:
   - FACP
   uses:
   - fire_alarm
-  - supervisor_alarm
   - water_flowrate_status
   - air_pressure_status
+  opt_uses:
+  - supervisor_alarm
```

Comment:
Apologies for the confusion regarding alarms. Since this type was already made with `supervisor_alarm` as required, we cannot move it to `opt_uses` - so please move this field back to the `uses` section. However, going forward, alarms should generally be listed under `opt_uses`.

Reply:
yes sometimes it is confusing. I'll revert back supervisor_alarm to uses section. thanks

Reply:
thank you